lib/twitter_web/templates/like/index.html.eex

```diff
@@ -0,0 +1,19 @@
+<h1>Listing Likes</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>likers</th>
+
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for l <- @likes do %>
+    <tr>
+      <td><%= l.user_id %></td>
```

Comment:
Except from Ecto queries or `i <- 1..10`, we usually avoid one-letter variable names since they decrease readability.
```suggestion
<%= for like <- @likes do %>
    <tr>
      <td><%= like.user_id %></td>
```
In other communities like golang, it is pretty standard practice to use cryptic variable names, but in Elixir it is not so common 😉

Reply:
ok, got it. 

Also in this file, I need to return the username searched by like.user_id, not the like.user_id itself. I thought I can do these things and vue and didn't spend any effort on the html files.

Reply:
you can do `like.user.username`, but you'll need to make sure you [preload](https://hexdocs.pm/ecto/Ecto.Query.html#preload/3) first 😉
Note: there are actually 2 ways to preload (using Query and using Repo), we try to use `Query` as long as it's possible.

Reply:
ok. thanks. 
I'm not sure where to display the list of likes, I don't think I can use the index file anymore, right?

Reply:
probably in the post detail page?
`for like <- @post.likes do`